pkg/collector/python/datadog_agent.go

```diff
@@ -203,3 +223,20 @@ func ObfuscateSQL(rawQuery *C.char, errResult **C.char) *C.char {
 	// memory will be freed by caller
 	return TrackedCString(obfuscatedQuery.Query)
 }
+
+// ObfuscateSQLExecPlan obfuscates the provided json query execution plan, writing the error into errResult if the
+// operation fails
+//export ObfuscateSQLExecPlan
+func ObfuscateSQLExecPlan(jsonPlan *C.char, normalize C.bool, errResult **C.char) *C.char {
+	obfuscatedJSONPlan, err := lazyInitObfuscator().ObfuscateSQLExecPlan(
+		C.GoString(jsonPlan),
+		normalize == C.bool(true),
```

Comment:
`bool(normalize)` doesn't work ?

Reply:
um... yes, looks like it does! Didn't think of trying that. Updated.